go/x509/x509.go

```diff
@@ -497,12 +530,27 @@ type Certificate struct {
 	// field is not populated when parsing certificates, see Extensions.
 	ExtraExtensions []pkix.Extension
 
+	// UnhandledCriticalExtensions contains a list of extension IDs that
```

Comment:
This is in the [upstream code](https://github.com/golang/go/blob/release-branch.go1.7/src/crypto/x509/x509.go#L525), so no change markers needed.